docs/api/navigation-history.md

```diff
@@ -9,6 +9,24 @@ Each navigation entry corresponds to a specific page. The indexing system follow
 
 ### Instance Methods
 
+#### `navigationHistory.canGoBack()`
+
+Returns `boolean` - Whether the browser can go back to previous web page.
+
+#### `navigationHistory.canGoForward()`
+
+Returns `boolean` - Whether the browser can go forward to next web page.
+
+#### `navigationHistory.canGoToOffset(offset)`
+
+* `offset` Integer
+
+Returns `boolean` - Whether the web page can go to `offset`.
```

Comment:
```suggestion
Returns `boolean` - Whether the web page can go to the specified `offset` from the current entry.
```

I liked the wording in the older API better because it was clearer about where the offset was from.